From cad4ff77d74a937fb97932a075dac4602fbcd998 Mon Sep 17 00:00:00 2001
From: Akihiro Suda <akihiro.suda.cz@hco.ntt.co.jp>
Date: Thu, 16 Dec 2021 18:27:42 +0900
Subject: [PATCH 61/62] configure.ac: Downgrade autoconf from 2.71 to 2.59

Autoconf 2.71 is not available for most distros yet: https://pkgs.org/search/?q=autoconf

Removal of AC_CHECK_INCLUDES_DEFAULT (introduced in 2.70) is safe,
according to https://www.gnu.org/software/autoconf/manual/autoconf-2.70/html_node/Default-Includes.html

Partially revert commit 7c4cf2f9002336086dc60f261a24aeac31cddf1b (PR 32)

Signed-off-by: Akihiro Suda <akihiro.suda.cz@hco.ntt.co.jp>
---
 configure.ac | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index e8edb1390..e21b5ae2a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,7 +1,7 @@
 #                                               -*- Autoconf -*-/
 # Process this file with autoconf to produce a configure script.
 
-AC_PREREQ([2.71])
+AC_PREREQ([2.59])
 AC_INIT([vde2],[2.3.2],[info@v2.cs.unibo.it])
 AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([foreign dist-bzip2 subdir-objects])
@@ -32,7 +32,6 @@ AC_CHECK_LIB([pcap], [pcap_open_dead],
   [add_pcap=no ; warn_pcap=yes])
 
 # Checks for header files.
-AC_CHECK_INCLUDES_DEFAULT
 AC_PROG_EGREP
 
 AC_HEADER_SYS_WAIT
-- 
2.35.0

